.travis.yml

```diff
@@ -35,20 +31,26 @@ before_install:
   - conda update --yes conda
 
 install:
+  # gjslint only runs in py27
+  - conda create --yes -n gjslint python=2.7 pip six
+  - conda run -n gjslint pip install https://github.com/google/closure-linter/archive/master.zip
+  - conda deactivate
```

Comment:
This command shouldn't be necessary anymore, since the `conda run` above handles activation and deactivation, inline.

Reply:
Thanks, removed!